eng/pipelines/release-cli.yml

```diff
@@ -764,17 +876,29 @@ stages:
 
 
               #### Windows
+              
+              PowerShell install
+
               ``````
               powershell -c "Set-ExecutionPolicy Bypass Process; irm '$urlBase/uninstall-azd.ps1' > uninstall-azd.ps1; ./uninstall-azd.ps1;"
               powershell -c "Set-ExecutionPolicy Bypass Process; irm '$urlBase/install-azd.ps1' > install-azd.ps1; ./install-azd.ps1 -BaseUrl '$urlBase' -Version '' -Verbose;"
               ``````
 
+              MSI install
+              ``````
+              powershell -c "irm '$msiReleaseUrl' -OutFile azd-windows-amd64.msi; msiexec /i azd-windows-amd64.msi /qn"
```

Comment:
Shouldn't the script i.e. _install-azd.ps1_ just do this itself? Seems more trouble to bifurcate instructions for the user. Packaging it as an MSI makes it installable from more locations e.g., chocolatey, scoop, and winget that we will do in another PR.

Reply:
We'll do that in a separate PR. I want to have an official release of the MSI before we put that into the install script. Otherwise we'd break the install script in a pretty fundamental way if it tried to download an MSI that didn't exist.